cromwell_tools/tests/test.sh

```diff
@@ -0,0 +1,4 @@
+docker build -t cromwell-tools:test ../..
```

Comment:
It would be better to add a comment here saying that this script can be only executed from within the current directory.